server/metadata/encoding/dictionary.go

```diff
@@ -133,8 +134,8 @@ func (r *reservedSubspace) reload(ctx context.Context, tx transaction.Tx) error
 			return api.Errorf(codes.Internal, "unable to deduce the encoded key from fdb key %T", allocatedTo)
 		}
 
-		r.allocated[ByteToUInt32(row.Value)] = allocatedTo.(string)
-		r.namespaceToId[allocatedTo.(string)] = ByteToUInt32(row.Value)
+		r.allocated[row.Data.(*internal.DictEncData).EncValue] = allocatedTo.(string)
```

Comment:
Type casting in multiple places doesn't look nice to me.
Can we create a helper method?
```
row.Data.ToDictEnc()
```
or 
```
GetDictEnc(row.Data)
```

Reply:
And same for Table and Raw.

Reply:
yeah agree - sure, let me think about it.